### PR TITLE
Week 4 Pull Request

### PR DESCRIPTION
**Pull request by AliannaFormoso**
# Week 4 `Pull request`

### CODE REVIEW

**Comment by AliannaFormoso** [comment]
# Week 4 `Lessons`
- Dayjs and Sass Blog
1. Installed `vite`, `dayjs`, and `sass`
2. Structured main folders and files
3. Added Html, first styles and js functionalities.

**Comment by AliannaFormoso** [comment]
# Week 4 `Lessons`
- Scss specifities
1. Added Scss files with more specific styles

**Comment by AliannaFormoso** [comment]
# Week 4 `Lessons`
- Dayjs and Sass Cvs
1. Installed `vite`, `dayjs`, and `sass`
2. Structured main folders and files
3. Added Html, first styles and js functionalities.

**Comment by AliannaFormoso** [comment]
# Week 4 `Lessons`
- Scss specifities CV
1. Added Scss files with more specific styles

File: student-cvs/alifm-2843/main.mjs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime.js";
+dayjs.extend(relativeTime);
+
+//* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
```
Comment (EzequielCaste) [inline student-cvs/alifm-2843/main.mjs:5]:
```suggestion
//* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
```

Just as this comment is suggesting, to **Toggle** the class, there is a method precisely for that in the **classList** property:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

File: student-cvs/alifm-2843/index.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+                </li>
+                <li>
+                    <a href="#acknowledgements">
+                        <i class="fa fa-fw fa-trophy"></i>
+                        Acknowledgements
+                    </a>
+                </li>
+             
+                <li>
+                    <a href="javascript:void(0);" class="icon">
```
Comment (EzequielCaste) [inline student-cvs/alifm-2843/index.html:53]:
```suggestion
                    <a href="#" class="icon">
```

can be replaced with a simple "#"

**Review by EzequielCaste — APPROVED** [review]
Review of Blog:
- Everything looks fine to me :+1: 
- One suggestion is to start using semantic HTML elements, for example, replace the `div`'s with `article` or `section`
- Check that the links on top, "Home, Beach, Country" don't have a value set to the `href` attribute, this causes the whole page to refresh. You can include a "#" as the value for `href` to avoid this behavior.

Review of CV
- All good (See comments)

You have my approval since your work meets the requirements for Week 4. good job! 🤗

File: student-cvs/alifm-2843/main.mjs (hunk 1 above)
Comment (AliannaFormoso) [reply student-cvs/alifm-2843/main.mjs:5]: Hi!
Great! I'll take a look of the article you recommended, and try to improve my code according to it.
Thank you for letting me know about it!

**Comment by AliannaFormoso** [comment]
> Review of Blog:
> 
> * Everything looks fine to me 👍
> * One suggestion is to start using semantic HTML elements, for example, replace the `div`'s with `article` or `section`
> * Check that the links on top, "Home, Beach, Country" don't have a value set to the `href` attribute, this causes the whole page to refresh. You can include a "#" as the value for `href` to avoid this behavior.
> 
> Review of CV
> 
> * All good (See comments)
> 
> You have my approval since your work meets the requirements for Week 4. good job! 🤗


Thank you so much for taking the time to review my PR!
 I'll apply all your suggestions since they really improve my work.
I appreciate it!

**Comment by AliannaFormoso** [comment]
# Week 4 `Review`
- Applied Reviewer's suggestions
1. Added semantic tags to blog page as well as addresses to the `href` attributes.
2. Modified function to toggle classes on CV page, as well as modified `href` for the icon.

**Comment by Arzeus17** [comment]
Todo óptimo!

**Review by Arzeus17 — APPROVED** [review]

**Comment by AliannaFormoso** [comment]
> Todo óptimo!

Muito obrigada!

**Comment by AliannaFormoso** [comment]
> # Congratulations @AliannaFormoso you did a great job.
> You have successfully completed the exercises for the week. 👏 Well done! Keep up the excellent work! 😁

Thank you 👍🏼

**Pull request merged**